align/pnr/checker.py

```diff
@@ -141,13 +142,18 @@ def check_place_on_grid(placement_verilog_d, concrete_name, opath):
     else:
         placement_dict = placement_verilog_d
 
+    # If we don't do the copy, then we get the 'flat_leaves" field filled in placement_verilog_d (can be large)
+    # We might want to use an auxillary data structure instead
+
+    placement_verilog_d_copy = copy.deepcopy(placement_verilog_d)
+
     placement = dict()
-    placement['leaves'] = {x['concrete_name']: x for x in placement_dict['leaves']}
```

Comment:
placement_dict is no longer being used and can be deleted.

Reply:
Yes.